services/src/main/java/io/druid/cli/CliBroker.java

```diff
@@ -105,6 +107,8 @@ public void configure(Binder binder)
             Jerseys.addResource(binder, ClientInfoResource.class);
             LifecycleModule.register(binder, QueryResource.class);
             LifecycleModule.register(binder, DruidBroker.class);
+            binder.bind(LookupReferencesManager.class).in(ManageLifecycle.class);
+            LifecycleModule.register(binder, LookupReferencesManager.class);
```

Comment:
@himanshug as you can see [here](https://github.com/druid-io/druid-api/blob/master/src/main/java/io/druid/guice/LifecycleModule.java#L68) registering is not enough to create the object, you need to either bind it to a scope or explicitly ask for it. maybe @cheddar can give a better explanation.